stock_warehouse_orderpoint_stock_info/__manifest__.py

```diff
@@ -0,0 +1,26 @@
+# Copyright 2016 OdooMRP Team
+# Copyright 2016 AvanzOSC
+# Copyright 2016 Pedro M. Baeza <pedro.baeza@tecnativa.com>
+# Copyright 2016 Serpent Consulting Services Pvt. Ltd.
+# Copyright 2017 Eficent Business and IT Consulting Services, S.L.
+# Copyright 2018 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Stock Warehouse Orderpoint Stock Info",
+    "version": "12.0.1.0.0",
+    "depends": [
+        "stock",
+    ],
+    "author": "OdooMRP team, "
+              "AvanzOSC, "
+              "Tecnativa, "
+              "Ecosoft, "
```

Comment:
You can maybe set you as author if you do a significant (not simple migration) contribution. Let just contributors line.